scrapi/base/schemas.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import unicode_literals
+
+from copy import deepcopy
+from dateutil.parser import parse
+
+from nameparser import HumanName
+
+
+def default_name_parser(names):
+    contributor_list = []
+    for person in names:
+        name = HumanName(person)
+        contributor = {
+            'prefix': name.title,
+            'given': name.first,
+            'middle': name.middle,
+            'family': name.last,
+            'suffix': name.suffix,
+            'email': '',
+            'ORCID': ''
+        }
+        contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def format_tags(all_tags):
+    tags = []
+    if isinstance(all_tags, basestring):
+        for taglist in all_tags.split(' '):
+            tags += taglist.split(',')
+
+    return list(set([unicode(tag.lower().strip()) for tag in tags if tag.lower().strip()]))
+
+
+def update_schema(old, new):
+    d = deepcopy(old)
+    for key, value in new.items():
+        if isinstance(value, dict) and old.get(key) and isinstance(old[key], dict):
+            d[key] = update_schema(old[key], new[key])
+        else:
+            d[key] = value
+    return d
+
+
+BASEXMLSCHEMA = {
+    "description": ('//dc:description/node()', lambda x: unicode(x.strip())),
+    "contributors": ('//dc:creator/node()', lambda x: default_name_parser(x.split(';'))),
+    "title": ('//dc:title/node()', lambda x: unicode(x.strip())),
+    "dateUpdated": ('//dc:dateEntry/node()', lambda x: unicode(x.strip())),
+    "id": {
+        "url": ('//dcq:identifier-citation/node()', lambda x: unicode(x.strip())),
```

Comment:
We might want to change this line in particular to something OAI specific - this namespace is only used in doepages